src/picosvg/svg.py

```diff
@@ -765,9 +765,10 @@ def _apply_gradient_translation(self, inplace=False):
                 x_prime = (r1 - c * y_prime) / a
 
                 # sanity check: a`(x`, y`) should be a(x, y)
-                p = affine.map_point((x, y))
+                # all our float brutality damages points; low tolerence sanity checks!
+                p = Point(r1, r2)
                 p_prime = affine_prime.map_point((x_prime, y_prime))
-                assert p.almost_equals(p_prime)
+                assert p.almost_equals(p_prime, tolerence=1e-1), f"{p} != {p_prime}"
```

Comment:
typo:

```suggestion
                assert p.almost_equals(p_prime, tolerance=1e-1), f"{p} != {p_prime}"
```

didn't the the relevant test fail?

Reply:
the misspelling tolerence->tolerance is in picosvg/geometric_types.py, in the `almost_equal` function and the `Point.almost_equal` method, and in the global variable `_DEFAULT_ALMOST_EQUAL_TOLERENCE`

Reply:
fixed